pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "notifications-utils"
-version = "52.3.3"
+version = "52.3.4"
```

Comment:
We might want to make this a major upgrade as this could break backward compat with other modules.